test/Dialect/FIRRTL/lower-types.mlir

```diff
@@ -1045,18 +1045,23 @@ firrtl.module private @is1436_FOO() {
 
 firrtl.circuit "NLALowering" {
   // Check if the NLA is updated with the new lowered symbol on a field element.
-  firrtl.hierpath @nla [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b]
-  // CHECK: firrtl.hierpath @nla_0 [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b_data]
-  // CHECK: firrtl.hierpath @nla [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b_ready]
-  firrtl.hierpath @nla_1 [@fallBackName::@test, @Aardvark::@test_1, @NLALowering]
-  firrtl.hierpath @nla_2 [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b2]
+  firrtl.hierpath @nla_b [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b]
+  // CHECK:      firrtl.hierpath @nla_b [@fallBackName::@test, @Aardvark::@test, @NLALowering::@b_valid]
```

Comment:
Isn't this invalid NLA ? `@NLALowering::@b_valid` doesn't have any annotation with `circt.nonlocal=@nla_b` . 
Maybe Out verifier should complain, but is ignoring this somehow.

Reply:
This is what this is trying to fix. The issue is that it's entirely legal to have a hierarchical path that is not used by an NLA, e.g., it could only be used by an `sv::VerbatimOp`.

Reply:
Ah, that makes sense. So, this should make it legal for any user.